dipy/direction/probabilistic_direction_getter.py

```diff
@@ -152,21 +158,28 @@ def from_shcoeff(klass, shcoeff, max_angle, sphere, basis_type=None,
 
         """
         pmf_gen = SHCoeffPmfGen(shcoeff, sphere, basis_type)
-        return klass(pmf_gen, max_angle, sphere, **kwargs)
+        return klass(pmf_gen, max_angle, sphere, pmf_threshold, **kwargs)
 
-    def __init__(self, pmf_gen, max_angle, sphere=None, **kwargs):
+    def __init__(self, pmf_gen, max_angle, sphere=None, pmf_threshold=0.1,
+                 **kwargs):
         """Direction getter from a pmf generator.
 
         Parameters
         ----------
         pmf_gen : PmfGen
-            Used to get probability mass function for choosing tracking
+            Used to get probability mass function for selecting tracking
             directions.
         max_angle : float, [0, 90]
             The maximum allowed angle between incoming direction and new
             direction.
+        mpmf_threshold : float, [0, 1.]
+            The maximum allowed angle between incoming direction and new
```

Comment:
Also, I think this shouldn't be here at all. There's no such parameter that I can see.